src/particles/plasma/PlasmaParticleContainer.cpp

```diff
@@ -12,6 +12,10 @@
 #include "utils/AtomicWeightTable.H"
 #include "utils/DeprecatedInput.H"
 #include "utils/GPUUtil.H"
+#include "utils/InsituUtil.H"
+#ifdef HIPACE_USE_OPENPMD
+#   include <openPMD/auxiliary/Filesystem.hpp>
+#endif
```

Comment:
Because later we need it when we call:

```
#ifdef HIPACE_USE_OPENPMD
    // create subdirectory
    openPMD::auxiliary::create_directories(m_insitu_file_prefix);
#endif
```

Note that this probably means that also the insitu-diags require openPMD to work.